### PR TITLE
Feature/protocol params#18

### LINKED ISSUE #18: [params] review protocol params
files: 
https://github.com/kowala-tech/kUSD/blob/master/params/protocol_params.go
https://github.com/kowala-tech/kUSD/blob/master/params/gas_table.go - bring the last gas prices from the ethereum codebase.

### PR DESCRIPTION
**Pull request by rgeraldes**
PR Closes #18
Params review.

### CODE REVIEW

File: kusd/downloader/downloader.go [hunk 1]
```diff
@@ -30,12 +29,12 @@ var (
 	MaxReceiptFetch = 256 // Amount of transaction receipts to allow fetching per request
 	MaxStateFetch   = 384 // Amount of node state values to allow fetching per request
 
-	MaxForkAncestry  = 3 * params.EpochDuration // Maximum chain reorganisation
-	rttMinEstimate   = 2 * time.Second          // Minimum round-trip time to target for download requests
-	rttMaxEstimate   = 20 * time.Second         // Maximum rount-trip time to target for download requests
-	rttMinConfidence = 0.1                      // Worse confidence factor in our estimated RTT value
-	ttlScaling       = 3                        // Constant scaling factor for RTT -> TTL conversion
-	ttlLimit         = time.Minute              // Maximum TTL allowance to prevent reaching crazy timeouts
+	MaxForkAncestry  = 3 * uint64(30000) // Maximum chain reorganisation (3 * params.EpochDuration - 30000)
```
Comment (yourheropaul) [inline kusd/downloader/downloader.go]: Why 30, 000? And Can we put this in a `const` somewhere so it's easy to change?

Comment (rgeraldes) [reply kusd/downloader/downloader.go]: reverted the change - added an issue to cover that field - https://github.com/kowala-tech/kcoin/issues/189 - Our epoch is shorter than Ethereum's.

File: kcoin/downloader/downloader.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ var (
 	MaxReceiptFetch = 256 // Amount of transaction receipts to allow fetching per request
 	MaxStateFetch   = 384 // Amount of node state values to allow fetching per request
 
-	MaxForkAncestry  = 3 * params.EpochDuration // Maximum chain reorganisation
+	MaxForkAncestry  = 3 * params.EpochDuration // Maximum chain reorganisation (3 * params.EpochDuration - 30000)
```
Comment (acroca) [inline kcoin/downloader/downloader.go]: The value in the comment is wrong. Also, why copy the value in the comment?

File: params/denomination.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package params
 
 const (
-	// These are the multipliers for ether denominations.
+	// These are the multipliers for kUSD denominations.
```
Comment (acroca) [inline params/denomination.go]: kcoin instead?

File: params/denomination.go [hunk 4]
```diff
@@ -1,7 +1,7 @@
 package params
 
 const (
-	// These are the multipliers for ether denominations.
+	// These are the multipliers for kCoin denominations.
```
Comment (acroca) [inline params/denomination.go]: `kcoin` better, no?

**Pull request merged**